command/parser/ast.go

```diff
@@ -150,7 +151,7 @@ func (b *Boolean) Capture(values []string) error {
 	} else if lv == "false" {
 		*b = false
 	} else {
-		return participle.Errorf(lexer.Position{}, "invalid value for 'Transient' field in source definition: %s", values[0])
+		return participle.Errorf(lexer.Position{}, "invalid value for boolean field in source definition: %s", values[0])
```

Comment:
Search and replace error?

Reply:
I changed this error message since there are now two boolean fields that can have invalid values, not just the `Transient` field

Reply:
So can we just pass in the field name to this method?

Reply:
Not a biggie - don't let it block merging this

Reply:
Ok, I'll merge it. This method implements an interface from an external library, which is why I didn't change the signature to include field name